megatron/arguments.py

```diff
@@ -641,6 +641,8 @@ def _add_training_args(parser):
                        'training if SIGTERM is received')
     group.add_argument('--tensorboard-dir', type=str, default=None,
                        help='Write TensorBoard logs to this directory.')
+    group.add_argument("--use-kernels-from-apex", action="store_true",
+                       help="use Apex kernels instead of Megatron")
```

Comment:
If a user builds apex from source, they already have the kernels to run everything.
So, its a waste to build kernels in this repo since they are just a copy of the ones in Apex.

Reply:
dropped this argument too